core/src/main/java/nl/nn/adapterframework/http/HttpSender.java

```diff
@@ -578,7 +643,7 @@ public static String handleMultipartResponse(String mimeType, InputStream inputS
 						charset = contentType.getCharset().name();
 
 					InputStream bodyPartInputStream = bodyPart.getInputStream();
-					result = Misc.streamToString(bodyPartInputStream, charset);
+					result = new Message(bodyPartInputStream, charset);
```

Comment:
Nu moet je wel oppassen: Als je dit result teruggeeft als result van de sender, dan wordt de stream pas gelezen als de method al geclosed is. Dan moet je dus of 1) die close uitstellen totdat de close van het readen van de stream of 2) deze inputstream naar een outputream schrijven.
Optie 1) is een beetje tricky, omdat het lastig is te garanderen dat die stream werkelijk altijd gelezen en geclosed wordt.  Het is niet zo safe als een try { ... } finally zeg maar.